src/platform/export/exportDialog.ts

```diff
@@ -1,5 +1,10 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Nice catch. Wonder if we need a rule for this. Not sure when it actually legally comes into play, but we do need this.

Reply:
I'm actually not sure we need this. Other microsoft repos don't always have this either, but we could write an eslint rule for this if we wanted to enforce.

Reply:
I think it's necessary, especially that when you copy some files from the repo to another (i.e., copying some helper files from VS Code to a different repo), you would want the copyright to come with it. We enforce this in our eslint rule https://github.com/microsoft/vscode/blob/0f648cd7dbafdbbbefda0f23c61c6c48a8d77619/.eslintrc.json#L94-L103